variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "environment" {
+  type        = string
+  description = "This is the environment in which the load balancer will be running"
+}
+
+variable "bucket_names" {
```

Comment:
best have validation https://stackoverflow.com/questions/50480924/regex-for-s3-bucket-name to prevent failing during runtime.

Reply:
As terraform's regex engine (re2 as mentioned [here](https://www.terraform.io/language/functions/regex) doesn't support lookahead, will have to find an equivalent expression